lib/interface/cli/commands/trigger/delete.cmd.js

```diff
@@ -0,0 +1,42 @@
+const debug = require('debug')('codefresh:cli:delete:trigger');
+const Command = require('../../Command');
+const CFError = require('cf-errors');
+const _ = require('lodash');
+const { trigger } = require('../../../../logic').api;
+const deleteRoot = require('../root/delete.cmd');
+
+const command = new Command({
+    command: 'trigger <event-uri> <pipeline>',
```

Comment:
Delete should work with the id/name of the created resource

`codefresh delete trigger TRIGGER_ID`

Reply:
There is no unique ID. It's is a link between trigger-event and pipeline. It's identified by trigger-event URI (unique) and pipeline id (unique). I think there is no need to create an additional ID. It's the combination of both that is unique.

Reply:
same as in trigger-event, there should be a unique name that will be possible to get/delete by it.

you can also support the easy and nice way you provided but the trigger should have a unique name